src/main/java/com/dynatrace/loadrunner/CommandLineParser.java

```diff
@@ -0,0 +1,56 @@
+package com.dynatrace.loadrunner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class CommandLineParser {
+
+	final public Map<String, String> arguments;
+
+	public CommandLineParser(String[] args) {
+
+		arguments = new HashMap<>();
+
+		for (int i = 0; i < args.length; i++) {
+			String key = null;
+			if (args[i].startsWith("-")) {
+				key = args[i].substring(1);
```

Comment:
shouldn't we already check if the `key` is valid and throw IllArgExc in case it's not?